test/CadlRanchProjects.Tests/resource-manager-models-resources.cs

```diff
@@ -32,7 +32,7 @@ public class ResourceManagerModelsResourcesTests: CadlRanchTestBase
         });
 
         [Test]
-        [Ignore("https://github.com/Azure/autorest.csharp/issues/4878")]
+        [Ignore("https://github.com/Azure/autorest.csharp/issues/4876")]
```

Comment:
so we changed this because we put the wrong issue link initially?

Reply:
I think both are needed in the first place, after this change, only the existing one left.